app/javascript/styles/components/Gallery.scss

```diff
@@ -7,38 +7,53 @@
     @include media(large) {
       grid-template-columns: repeat(3, 1fr);
       width: 870px;
+      -ms-grid-columns: 1fr 1fr 1fr;
```

Comment:
If doing these starts to get cumbersome/messy you can look at using [autoprefixer](https://github.com/postcss/autoprefixer).

Reply:
I started this by trying to get autoprefixer up and running, but was having more trouble than I thought it was worth for now.